### DISCUSSION TITLE
Is it possible for a non-developer to create a new rss feed from a site?

### DISCUSSION THREAD

**Discussion by ghost** [DIYgod/RSSHub#14350] [Q&A] [answered]
I want to add a nsfw blog to my rss reader. I am not a developer. I looked over the docs and I am completely lost on how to do it.

The nsfw blog I want to add used to have an rss feed, but they removed it. rss radar only finds a comments feed on the site now, but I don't want that. I want an rss feed of the blog posts / articles / main links (whatever they are called). Is it possible for a non-dev to set up one?

I've looked through the docs. The nsfw blog I am trying to follow seems to be a wordpress blog, so I found this route:  /blogs/wordpress/:domain/:https? and when I enter the domain of the nsfw blog, I get an error: "Error message: This RSS is disabled unless 'ALLOW_USER_SUPPLY_UNSAFE_DOMAIN' is set to 'true'."

I guess that means it can't work?

**Comment by TonyRL** [comment, marked as answer]
You need a [self hosted](https://docs.rsshub.app/install) instance of RSSHub with `ALLOW_USER_SUPPLY_UNSAFE_DOMAIN` set to `true`. The default is false for obvious security reason.

**Comment by MengyangGao** [comment]
It is possible, but maybe not in the current RSSHub project. Currently, I found three solutions which can match your requirements. The first to recommend is an open source project too, [feedless](https://github.com/damoeb/feedless). The developer makes a GUI for picking the element user wants on the webpage. The next is [RSS anything](https://rss.diffbot.com), from a company called [diffbot](https://www.diffbot.com). The developer loves rss so he/she makes the rss finder function of his company free(for now).  The third one is [FetchRSS](https://fetchrss.com), a commercial service, the free plan has limits. I am learning to make contributions on rsshub too, but lack of front end knowledge. These three are someway alternatives. And I sincerely hopes that oneday RSSHub can have the GUI for picking elements users want, just like feedless.
